cirq-google/cirq_google/json_test_data/InternalTag.json

```
@@ -0,0 +1,6 @@
+{
+    "cirq_type": "InternalTag",
+    "name": "IdleWaitTag",
+    "package": "internal.module",
+    "phase_match": true,
+    "pad_ns": 20.0}
```

In case we care about json format symmetry -
```suggestion
    "pad_ns": 20.0
}
```

---

Done.